platform/windows/detect.py

```diff
@@ -390,7 +390,10 @@ def configure_msvc(env: "SConsEnvironment", vcvars_msvc_config):
         env.AppendUnique(CPPDEFINES=["R128_STDC_ONLY"])
         env.extra_suffix = ".llvm" + env.extra_suffix
 
-    env["MAXLINELENGTH"] = 8192  # Windows Vista and beyond, so always applicable.
+    # In case the command line to AR (lib) is too long, use a response file.
+    env["ARCOM_ORIG"] = env["ARCOM"]
+    env["ARCOM"] = "${TEMPFILE('$ARCOM_ORIG', '$ARCOMSTR')}"
+    env["TEMPFILESUFFIX"] = ".rsp"
```

Comment:
That's likely not necessary at all, for MSVC Scons seems to have `TEMPFILE` enabled for `AR` by default. Removing the `MAXLINELENGTH` should be sufficient to fix build issues.